parlai/agents/drqa/drqa.py

```diff
@@ -57,7 +57,6 @@ def add_cmdline_args(argparser):
             default=True,
             help='Use only words found in provided embedding_file',
         )
-        group.set_defaults(dict_tokenizer='spacy')
```

Comment:
hmm the docstring still says we're using the spaCy tokenizer

Reply:
thx!